forgerock-authenticator/CHANGELOG.md

```diff
@@ -4,3 +4,14 @@
 
 #### Added
 - Initial release of ForgeRock Authenticator for Flutter
+
+## [1.1.0]
```

Comment:
The new features and releases should appear on top of the page... at least all our other repos follow this convention.

Reply:
Nice catch! Done